components/kyma-environment-broker/internal/fixture/fixture.go

```diff
@@ -17,6 +17,7 @@ const (
 	ServiceId                   = "47c9dcbf-ff30-448e-ab36-d3bad66ba281"
 	ServiceName                 = "kymaruntime"
 	PlanId                      = "4deee563-e5ec-4731-b9b1-53b42d855f0c"
+	TrialPlan                   = "7d55d31d-35ae-4438-bf13-6ffdfa107d9f"
```

Comment:
nit: you can use plan IDs consts from broker package:
https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/broker/plans.go#L13

Reply:
I actually tried that before but it creates an import cycle, I think that is the reason why `PlanID` is also re-defined here one line higher